TodoList/TodoList.Api.Tests/Utils/ExtensionItemController.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Web.Http;
+using TodoList.Api.Controllers;
+
+namespace TodoList.Api.Tests.Utils
+{
+    public static class ExtensionItemController
+    {
+        public static async Task<HttpResponseMessage> ExecuteAction(
```

Comment:
Is this method really `ItemController` bound? Is it a rather generic method that could be used with any controller? What about making it method that accepts any controller while remaining strongly typed? 😄 [note: `ControllerExtensions` would then be a good class/file name 😉)